draft-ietf-acme-acme.md

```diff
@@ -210,10 +209,8 @@ The server then validates the challenges to check that the client has
 accomplished the challenge.
 
 Once the validation process is complete and the server is satisfied that the
-client has met its requirements, the server can either proactively issue the
-requested certificate or wait for the client to request that the application be
-"finalized", at which point the certificate will be issued and provided to the
-client.
+client has met its requirements, the server MUST proactively issue the
```

Comment:
Don't think the word "proactively" is needed.  It was useful in the discussion, but not so much now.

Let's drop the "MUST".  It's covered below; this is just an overview.